corehq/apps/users/views/__init__.py

```diff
@@ -1136,6 +1138,10 @@ def post(self, request, *args, **kwargs):
                 primary_location_id = data.pop("primary_location", None)
                 data["primary_location"] = (SQLLocation.by_location_id(primary_location_id)
                                         if primary_location_id else None)
+                assigned_location_ids = data.pop("assigned_locations", None)
+                data["assigned_locations"] = [SQLLocation.by_location_id(assigned_location_id)
+                                        if assigned_location_id else None
```

Comment:
Are these ids are ever expected to be falsy, or is this just defensive programming?

Reply:
they're not expected to be false so just defensively programming. But I really do just want to avoid any "None" values [6816b2f](https://github.com/dimagi/commcare-hq/pull/34466/commits/6816b2f0053a331465c4ce141129933d1667da6f)